f3/types.go

```diff
@@ -1,11 +1,50 @@
 package f3
 
 import (
+	"math/big"
 	"strings"
 )
 
 type ActorID uint64
 
+type StoragePower big.Int
```

Comment:
Make this a type alias and then you can remove all the wrapper methods.

`type StoragePower  = big.Int`

This is slightly less "safe" in that it doesn't distinguish between different things a big.Int might be representing, but I think it's fine here (and Lotus does too).

Reply:
Ah ok, I thought about this too but I did not do it precisely to distinguish between different things. But agreed that helps remove boilerplate. Done.